README.md

```diff
@@ -48,43 +48,35 @@ Changing this property from its default will automatically reset `recycleElement
 
 ### `updateElement` property
 
-Type: `function(child: Element, itemIndex: number)`
+Type: `function(child: Element, item: any, itemIndex: number)`
 
-Set this property to configure the virtual scroller with a function that will update the element with data at a given index.
+Set this property to configure the virtual scroller with a function that will update the element with data from a given item at the specified index.
 
 This property is invoked in these scenarios:
 
 * The user scrolls the scroller, changing which items' elements are visible. In this case, `updateElement` is called for all of the newly-visible elements.
-* The developer changes the `totalItems` property.
-* The developer calls `requestReset()`, which will call `updateElement` for all currently-visible elements. See [below](#data-manipulation-using-requestreset) for why this can be useful.
+* The developer changes the `itemSource` property.
+* The developer calls `itemsChanged()`, which will call `updateElement` for all currently-visible elements. See [below](#data-manipulation-using-itemschanged) for more on this.

```

Comment:
What if there is no recycling, should it then not update all elements?

Reply:
the elements should be updated in that case too, as currently visible data might have changed.